docs/packages.md

```diff
@@ -0,0 +1,45 @@
+# Packages
+
+[up](README.md)
+
+During an image build, OpenEmbedded builds all software into packages, which are then
+combined into a rootfs. One benefit of this is you can install packages later into a
+running system, much like you would a desktop distribution like Debian, Ubuntu, Arch,
+etc. OE supports the RPM, DEB, and IPK package formats, and the format can be set using the
+[PACKAGE_CLASSES](https://www.yoctoproject.org/docs/2.6/mega-manual/mega-manual.html#var-PACKAGE_CLASSES)
+variable.
+
+## Installing packages from a local build
+
+You can install packages directly from your local OE build onto a target system with the
+following procedure:
+
+- packaging information must be included in the target image:
+  - `IMAGE_FEATURES += "packages-management"`
+- a ssh server must be enabled in the target image:
+  - `IMAGE_FEATURES += "ssh-server-dropbear"`
+- target system just be on the same network as your build computer.
+- run `yoe_setup_feed_server <target ip address>`. This configures the target to fetch packages
+  from your build computer. Alternatively you can set the MACHINE_IP environment variable.
+- run `yoe_feed_server`. This starts a web server that serves packages.
+- on target system, run:
+  - `opkg update`
+  - `opkg list` (list packages available)
+  - `opkg install python3` (install python3)
+
+If the package is not built yet, simply bitbake it and rerun `yoe_feed_server`
+(required to rebuild the package index). You can then `opkg install` the package
+on the target system.
```

Comment:
this looks pretty detailed and good.  We need to mention that one needs to refresh the package index after building a new package with 
```bitbake package-index```

otherwise opkg update will not get new index updated and hence not see the newly built package.

Reply:
yoe_feed_server runs bitbake package-index, so I think we are covered here.

Reply:
that would mean one has to stop server, build the package, restart the server and then do opkg update, I generally keep the server running once launched